ui/src/config.js

```diff
@@ -71,59 +71,70 @@ export const config = {
           type: 'vocabulary',
           vocabularyType: 'document_accelerators',
           default: '',
-        },
-        'unit:curated_relation': {
-          label: 'Curated Relation',
-          type: 'boolean',
-          default: false,
+          line: 1,
         },
         'unit:experiment': {
           label: 'Experiment',
           type: 'vocabulary',
           vocabularyType: 'document_experiments',
           default: '',
+          line: 1,
```

Comment:
Hmm, I understand your point but if we add another component that wants to display them differently, it can just omit the `line` attribute. When I added this `line` attribue, I had in mind to make it easy to any user of invenio to organize the extra fields easily when creating their app. As I understand it, we have this configuration object because for each invenio based project we will have different extra fields, so we should be able to organize the fields acordingly to the project. In our case, in `cds` it makes sense to have 3 fields in the first line but maybe others want only 1 field per line. This was the motivation behind the change, it doesn't feel right to me to provide a static layout for dynamic fields.

Reply:
Agree with your comments, but it does not feel right either to put presentation in configuration: if you put an "order" key, then it means that an order is important when presenting, but it is not defining how you present it.
If you put `line`, you are really meaning that these will be show in lines, and not cols, buttons, labels, etc...